asllib/doc/Expressions.tex

```diff
@@ -1232,95 +1221,28 @@ \subsubsection{Comments}
 initializer expression.
 \lrmcomment{This is related to \identi{MJWM}.}
 
-\subsubsection{TypingRule.ESliceOrEGetArrayError\label{sec:TypingRule.ESliceOrEGetArrayError}}
+\subsubsection{TypingRule.ESliceError\label{sec:TypingRule.ESliceError}}
```

Comment:
You need to apply the renaming in `Typing.ml`
(`|: TypingRule.ESliceOrEGetArrayError`) and in `Instrumantation.ml`